### PR TITLE
Insects in the sky.

### PR DESCRIPTION
**Pull request by fluxsauce**

### CODE REVIEW

File: NOVEL.md [hunk 1]
```diff
@@ -1 +1,3 @@
-It was a sunny May day
+It was a sunny May day. The insects blackened the sky yet again with their
```
Comment (ajpierce) [inline NOVEL.md]: But if the sky is blackened, is it still a sunny day?

Comment (fluxsauce) [inline NOVEL.md]: Good point, clarified.

**Comment by Ghoughpteighbteau** [comment]
That bodes ill, a plot hole in the first two sentences. :hamster:

**Comment by why-el** [comment]
Care to rebase? :D

**Comment by fluxsauce** [comment]
:-( Will revisit after dinner.

**Comment by Ghoughpteighbteau** [comment]
I included this scene as part of #9

**Pull request closed without merging**